client/post-editor/style.scss

```diff
@@ -1,10 +1,18 @@
+/** @format */
+
+@import url( '//s1.wp.com/wp-includes/css/dashicons.css?v=20150727' );
```

Comment:
what's the scoop here?

Reply:
I'm not exactly sure but it came in at https://github.com/Automattic/wp-calypso/commit/a538e54e31dbe64458b406c7b56d1e6519063e2f. Before that it was placed in the HTML templates which are rendered on the server. It's used for displaying icons in TinyMCE.